client/dashboard/profile-wizard/steps/business-details.js

```diff
@@ -23,6 +24,8 @@ import withSelect from 'wc-api/with-select';
 import { recordEvent } from 'lib/tracks';
 import { formatCurrency } from '@woocommerce/currency';
 
+const wcAdminAssetUrl = getSetting( 'wcAdminAssetUrl', '' );
```

Comment:
In my PR I was able to remove this entirely by removing the images in favor of embedded SVGs. May want to do this also - it seemed much cleaner than passing paths around.

Reply:
Agreed, but I think it should be done in its own pull rather than slipped in with a larger refactor.  Thoughts?

Reply:
If it makes sense to nuke a setting completely it would save needing to rewrite bits twice? I don't mind. `wcAdminAssetUrl` was the one I think that only had usage in a few places, one of which was the component library (which makes no sense since those components should not have wc admin depdendency).

Reply:
Ya I understand the comment about rewriting bits twice, however I think there's value in differentiating between refactors preserving existing behaviour and pulls that change behaviour or enhance by nuking things.  Especially when it comes to having record of that change in the git history (which would be lost with the squash commit for this pull).

So I still think we should do this in it's own pull (which also might address other places where there's unnecessary dependencies).

Reply:
I agree its best to do it separately for a simpler record of the changes and why.

Reply:
https://github.com/woocommerce/woocommerce-admin/issues/2943